yogstation/code/datums/traits/negative.dm

```diff
@@ -5,4 +5,4 @@
 /datum/quirk/blooddeficiency/check_quirk(datum/preferences/prefs)
 	if(prefs.pref_species && (NOBLOOD in prefs.pref_species.species_traits)) //can't lose blood if your species doesn't have any
 		return "You don't have blood as [prefs.pref_species]!"
-	return FALSE
+	return ..()
```

Comment:
Why `return ..()`

Reply:
I was going to add a generic blacklist and it might be useful if we need some generic quirk denial things later